diff --git a/dist/cjs/modules/createDOMProps/index.js b/dist/cjs/modules/createDOMProps/index.js
index d643013..df3a41d 100644
--- a/dist/cjs/modules/createDOMProps/index.js
+++ b/dist/cjs/modules/createDOMProps/index.js
@@ -805,9 +805,7 @@ var createDOMProps = (elementType, props, options) => {
   }
 
   // Resolve styles
-  if (pointerEvents != null) {
-    (0, _warnOnce.warnOnce)('pointerEvents', "props.pointerEvents is deprecated. Use style.pointerEvents");
-  }
+
   var _StyleSheet = (0, _StyleSheet2.default)([style, pointerEvents && pointerEventsStyles[pointerEvents]], (0, _objectSpread2.default)({
       writingDirection: 'ltr'
     }, options)),
diff --git a/dist/modules/createDOMProps/index.js b/dist/modules/createDOMProps/index.js
index 18f5e8b..4e3494c 100644
--- a/dist/modules/createDOMProps/index.js
+++ b/dist/modules/createDOMProps/index.js
@@ -10,9 +10,8 @@ var _excluded = ["aria-activedescendant", "accessibilityActiveDescendant", "aria"];
  * 
  */
 
-import AccessibilityUtil from '../AccessibilityUtil';
 import StyleSheet from '../../exports/StyleSheet';
-import { warnOnce } from '../warnOnce';
+import AccessibilityUtil from '../AccessibilityUtil';
 var emptyObject = {};
 var hasOwnProperty = Object.prototype.hasOwnProperty;
 var isArray = Array.isArray;
@@ -801,9 +800,6 @@ var createDOMProps = (elementType, props, options) => {
   }
 
   // Resolve styles
-  if (pointerEvents != null) {
-    warnOnce('pointerEvents', "props.pointerEvents is deprecated. Use style.pointerEvents");
-  }
   var _StyleSheet = StyleSheet([style, pointerEvents && pointerEventsStyles[pointerEvents]], _objectSpread({
       writingDirection: 'ltr'
     }, options)),
diff --git a/src/modules/createDOMProps/index.js b/src/modules/createDOMProps/index.js
index 4144dbc..73948c1 100644
--- a/src/modules/createDOMProps/index.js
+++ b/src/modules/createDOMProps/index.js
@@ -7,9 +7,8 @@
  * @noflow
  */
 
-import AccessibilityUtil from '../AccessibilityUtil';
 import StyleSheet from '../../exports/StyleSheet';
-import { warnOnce } from '../warnOnce';
+import AccessibilityUtil from '../AccessibilityUtil';
 
 const emptyObject = {};
 const hasOwnProperty = Object.prototype.hasOwnProperty;
@@ -873,12 +872,6 @@ const createDOMProps = (elementType, props, options) => {
   }
 
   // Resolve styles
-  if (pointerEvents != null) {
-    warnOnce(
-      'pointerEvents',
-      `props.pointerEvents is deprecated. Use style.pointerEvents`
-    );
-  }
 
   const [className, inlineStyle] = StyleSheet(
     [style, pointerEvents && pointerEventsStyles[pointerEvents]]
   );